extensions/integration-tests/package.json

```diff
@@ -45,16 +45,17 @@
     ]
   },
   "scripts": {
+    "fmktest": "mocha -r ts-node/register --watch-extensions tsx,ts -u tdd test/**/*.test.ts",
     "vscode:prepublish": "node ../../node_modules/gulp/bin/gulp.js --gulpfile ../../build/gulpfile.extensions.js compile-extension:vscode-colorize-tests ./tsconfig.json",
     "postinstall": "node ./node_modules/vscode/bin/install"
   },
   "devDependencies": {
-    "@types/node": "7.0.43",
-    "@types/chai": "3.4.34",
+    "@types/node": "^10.12.12",
```

Comment:
This is to make these consistent with the devDependencies in the root package.json.